ci-operator/config/RedHatQE/interop-testing/RedHatQE-interop-testing-main__ocp-cnv-odf-4.11.yaml

```diff
@@ -34,6 +42,8 @@ tests:
     test:
     - ref: interop-tests-deploy-cnv
     - ref: interop-tests-deploy-odf
+    - ref: cucushift-installer-check-cluster-health
```

Comment:
Here should be `- chain: cucushift-installer-check-cluster-health`

Reply:
thx, I changed the line